metrics_activities_exports.go

```diff
@@ -0,0 +1,51 @@
+package chartmogul
```

Comment:
```
Your code does not pass gofmt in 1 place. Go fmt your code!
```
May be some whitespace issue. Try formatting with gofmt.

Reply:
yup, done!